test/CMakeLists.txt

```diff
@@ -171,8 +167,7 @@ if (FMT_PEDANTIC AND NOT WIN32)
     "-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}"
     "-DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}"
     "-DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}"
-    "-DFMT_DIR=${CMAKE_SOURCE_DIR}"
-    "-DSUPPORTS_USER_DEFINED_LITERALS=${SUPPORTS_USER_DEFINED_LITERALS}")
```

Comment:
``CXX_STANDARD_FLAG`` is not defined in ``test/compile-error-test/CMakeLists.txt``

Suggestion:
``+    "-DCXX_STANDARD_FLAG=${CXX_STANDARD_FLAG}"``